### PR TITLE
[#76] Add multiple choice UI [Last part]

### LINKED ISSUE #76: [UI] As a user, I can see different Answer choices
## Why

We need to display Answer choice depending on the type from the API.
 
## Who Benefits?
Prepare all the UI for all different choices:
image

### PR DESCRIPTION
**Pull request by sleepylee**
Closes #76

## What happened 👀
Add multi-choice UI.
 
## Insight 📝

- Basic list implementation with choice stored as selected states.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/13435717/125420159-fe8febf0-28dc-4906-8cdf-a92efcf227b8.mp4

Text size has been reduced in the latest commit:

### CODE REVIEW

File: lib/pages/ui/multiple_choice_rating.dart [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  final List<String> choices;
+
+  const MultipleChoiceRating({Key key, this.choices}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _MultipleChoiceRatingState();
+}
+
+class _MultipleChoiceRatingState extends State<MultipleChoiceRating> {
+  final _selectedIndex = <int>[];
```
Comment (manh-t) [inline lib/pages/ui/multiple_choice_rating.dart:14]: `_selectedIndexes`

File: lib/pages/ui/multiple_choice_rating.dart [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                              : Colors.white12),
+                    ],
+                  ),
+                  _shouldDrawDivider(index)
+                      ? const Divider(
+                          color: Colors.white,
+                          height: 30,
+                          thickness: 1,
+                        )
+                      : SizedBox.shrink()
```
Comment (manh-t) [inline lib/pages/ui/multiple_choice_rating.dart:77]: I think we can reduce the redundant widget by changing the syntax of condition a bit
```suggestion
                  if _shouldDrawDivider(index)
                      const Divider(
                          color: Colors.white,
                          height: 30,
                          thickness: 1,
                        )
```

**Comment by sleepylee** [comment]
thanks @manh-t 🙏  all updated

**Review by manh-t — APPROVED** [review]

**Review by markgravity — APPROVED** [review]
LGTM 🤖

**Pull request merged**